index.js

```diff
@@ -281,7 +281,7 @@ module.exports = {
     }],
     // 'one-var-declaration-per-line': 0,
     // 'operator-assignment': 0,
-    // 'operator-linebreak': 0,
+    'operator-linebreak': ['error', 'after'],
```

Comment:
Can you make this `2` to be consistent with the other errors?

Reply:
I assumed that the variant doesn't matter, because I saw the `'error'` form in two other places and `'error'` also felt more approachable to me than `2` when I started to dive into ESLint :).

I changed the other `'error'` occurences to `2` in order to maintain the consistency across the whole file. I hope you don't mind ;)

Reply:
I agree `'error'` is more approachable. I'm not sure why that wasn't the original choice, but I'd be open to changing it.